docs/analytics.md

```diff
@@ -4,27 +4,31 @@ Email Alert API does not currently have many automated means to generate
 analytics and data insights. This document serves as a guide to understand
 how to produce this information.
 
-## How data is organised
+## Available data
 
-Most of the data in this application is stored in the database for perpetuity,
-thus for most cases querying the database through Rails console or through
-PostgreSQL is appropriate. However as this application creates such a large
-volume of emails these are instead archived with minimal data in
-[Amazon S3](https://aws.amazon.com/s3/) and can be queried
-with [Amazon Athena][athena].
+We retain most unused data in the database for one year. This excludes emails
+(performance reasons) and unused subscriber lists (pollutes analytics) which
+are deleted after a week. More data isn't retained as we have no evidence of
+needing it.
 
-Once an email has been sent or is no longer being sent (such as a
-permanent failure) the record will be archived in a reduced form and sent to
-Amazon S3 in hourly batches. After 14 days the email will be removed from the
-database. Thus for recent information the `emails` database table can be used
-otherwise the email archive should be queried.
+See [ADR-7] for the most up to date data retention strategy. An overview of our
+data cleanup mechanisms can also be found [here][cleanup mechanisms].
```

Comment:
there's a few minor issues across these paragraphs that I think could be worked out with some rephrasing:

- unusued data - seems a bit weird,
- pollutes analytics - seems exotic
- up to date - seems time based
- should try avoid using a link on "here" if you can, seems less natural than summarising something with a link to wider info.

I'd try something like

We have established a [data retention strategy][adr-7] for Email Alert API. Emails are stored in the system for 7 days after they are sent. Subscriber lists that have never had a subscriber are also removed after 7 days. When a subscriber no longer has any subscriptions [their personally identifiable information](https://github.com/alphagov/email-alert-api/blob/master/docs/data-cleanup-mechanisms.md#nullifying-email-addresses) is removed after 28 days. Past subscriptions, content changes, messages and digest run information are stored for 1 year.

Reply:
I've taken inspiration from your suggestion and restructured the paragraphs. 

I wasn't really keen on adding individual models to this description as there is quite a few more than what you've listed and I think it's sufficient to just name out the ones that are different to the one year retention rule, if they really want to dig into the specifics they can look at the ADR.